demo/ios/ForegroundApp/ForegroundApp/ContentView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright 2018-2025 Picovoice Inc.
```

Comment:
```suggestion
//  Copyright 2025 Picovoice Inc.
```
New files don't require year range